securesignals-gma/build.gradle

```diff
@@ -0,0 +1,30 @@
+plugins {
+    id 'com.android.library'
+    id 'org.jetbrains.kotlin.android'
+}
+
+apply from:  rootProject.file("$rootDir/common.gradle")
+
+android {
+    namespace 'com.uid2.securesignals.gma'
+    defaultConfig {
+        minSdk 19
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+        }
+    }
+}
+
+dependencies {
+    implementation project(path: ':sdk')
+
+    // We only compile against the GMA SDK as the exact version being used should be defined by the consuming
+    // application.
+    compileOnly 'com.google.android.gms:play-services-ads:21.5.0'
```

Comment:
We'll need to confirm with Google what minimum version we need to support.  iOS is using GMA 10.x+

https://maven.google.com/web/index.html#com.google.android.gms:play-services-ads

Reply:
👍